maintainers/maintainer-list.nix

```
@@ -14099,6 +14099,8 @@
     github = "RaitoBezarius";
     githubId = 314564;
     name = "Ryan Lahfa";
+    supportedArchitectures = [ "x86_64-linux" "riscv64-linux" ];
+    limitedSupportedArchitectures = [ "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

2 suggestions i have

- `architectures` -> `platforms` to be more consistent with `meta.platforms`, also architecture is sometimes referred to only the `x86_64` and `riscv64` part, but not the kernel
- `limitedSupportArchitectures` feels a bit arbitrary and verbose, having a list of list allows users to have more than 2 tiers of support 

```suggestion
    supportedPlatforms = [
      [ "x86_64-linux" "riscv64-linux" ]
      [ "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ]
    ];
```

---

limitedSupportedArchitectures comes from `nixos/release-*.nix`  BTW.

---

Platforms follow the tier system. We can replay them here somehow.

---

That doubly-nested listing is rather hard to understand, and I don't think we need to use the same tiering from https://github.com/NixOS/rfcs/pull/46. How about

```nix
primaryPlatforms = [ ... ];
secondaryPlatforms = [ ... ];
```

---

That doubly-nested listing is rather hard to understand, and I don't think we need to use the same tiering from https://github.com/NixOS/rfcs/pull/46. How about

```nix
primaryPlatforms = [ ... ];
secondaryPlatforms = [ ... ];
```

---

Maybe even primary/available. I do use _a bit_ of Nix-managed stuff on `aarch64-linux` daily… but really not enough to have primary-platform competence there.

---

How about primary/supported? (Maybe with a check that the former is a sub set of the latter (Speaking of which, that maintainers list doesn't seem to be type/sanity checked anywhere?))

---

I like primary/supported and primary/available, though I wouldn't repeat the primary ones in supported/available.